sequencing/ATACSeq/README.md

```diff
@@ -195,33 +196,26 @@ This script will detect possible DNA contamination in order to ensure high quali
   - `SWITCH`: Indentify only mismatched genotype samples and output potential switches and seach for alternative genotype.
 
 
-#### Still to be developed ####
-
-
 ### 7. Group peak calling
 
- 7.0) `sbatch --array=<number of batch jobs> ATACSeq/jobSubmission/7_helperTools.sh (project directory) [GROUPS] [STEPS]`
- 
-This script includes several MACS functionalities/tools and samtools to perform different functionalities, such as produce samples with a subset of reads or create pseudo-replicates. 
-
+ `sbatch ./jobSubmission/7_batchPeakCallingByGroup.sh (project directory) [STEPS] [GROUP]`
 
- 7.1) `sbatch --array=<number of batch jobs> ATACSeq/jobSubmission/7_1_batchPeakCallingByGroup.sh (project directory) [GROUPS] [STEPS]`
-
-This script repeats peak calling but on groups of samples, which can be grouped by cell fraction or if they have passed the first stage of quality control.  
+This script groups samples per cell type, perform group level peak calling and gets read counts in peaks called. Samples used are those that have passed previous quality control stages 1 and 2. Cell fractions to which samples belong to need to be specified in the *config.txt* file.
```

Comment:
typo: **performs**

Also thanks for keeping the README up to date. Other parts of the repository should follow suit.